config/locations/holding_locations.json

```diff
@@ -3281,7 +3280,7 @@
         "delivery_locations": [
             "QT",
             "QA",
-            "PN",
+            "PT",
```

Comment:
`lewis$pn`

Reply:
This is the change to make, retain the other delivery locations as this is a recap location and sometimes they need to be requested by staff to those other locations.